package/metadata.json

```diff
@@ -28,11 +29,9 @@
 		"Name[fr]": "Command Output",
 		"Name[nl]": "Opdrachtuitvoer",
 		"Name[pt_br]": "Saída de Comandos",
-		"ServiceTypes": [
-			"Plasma/Applet"
-		],
-		"Version": "14",
+		"Version": "13",
 		"Website": "https://github.com/aricaldeira/plasma-applet-commandoutput"
 	},
+	"X-Plasma-API": "declarativeappletscript",
```

Comment:
Pretty sure this was dropped too.

Reply:
Memory was correct.

* https://invent.kde.org/plasma/plasma-desktop/-/commit/d016cae56387ee481a68c11d12e76e3bb0fd3999
* https://invent.kde.org/plasma/libplasma/-/merge_requests/906